docs/getting-started.mdx

```diff
@@ -5,38 +5,152 @@ description: Quick start guide for Xata's PostgreSQL platform
+## 3. Create a branch
 
-## Additional resources
+Within your project, create your first branch (e.g., `main`). A branch is a Postgres cluster—a collection of one or more Postgres instances, which can include a primary and optional replicas. When creating your main branch, you can choose the Postgres version, region, and instance size to fit your needs. This branch can serve as your production branch or a staging environment. You can branch off `main` at any time to create isolated development or test environments.
 
-Want to dive in further? Check out these pages.
+![Create branch page](assets/images/getting-started/create-branch.png)
 
-- [Architecture overview](/architecture) - Deep dive into Xata's technical architecture
-- [Core concepts](/core-concepts) - Learn about branching, staging, and data anonymization
-- [Command line interface](/cli) - Complete CLI reference
-- [GitHub actions](/github-actions) - Automate your development workflow
+## 4. Add sample xata (data)
+
+Navigate to the **Queries** section in your project and run the following SQL to create sample tables and data:
+
+```sql
+CREATE TABLE products(id SERIAL PRIMARY KEY,name TEXT,price NUMERIC(7,2));
+CREATE TABLE orders(id SERIAL PRIMARY KEY,created_at TIMESTAMPTZ DEFAULT now());
+CREATE TABLE order_items(order_id INT REFERENCES orders(id),product_id INT REFERENCES products(id),qty INT,PRIMARY KEY(order_id,product_id));
+INSERT INTO products(name,price) SELECT LEFT(md5(i::text),8),(random()*90+10)::numeric(7,2) FROM generate_series(1,10)i;
+WITH o AS (INSERT INTO orders DEFAULT VALUES RETURNING id) INSERT INTO order_items(order_id,product_id,qty) SELECT o.id,pid,(1+floor(random()*3))::int FROM o,(SELECT id pid FROM products ORDER BY random() LIMIT 5)p;
+```
+
+![Queries/SQL editor page](assets/images/getting-started/queries.png)
+
+## 5. View your branch schema and enable schema history
+
+Go to the **Branches** section and select your branch (e.g., `main`). Here you can view your schema and enable schema history for zero-downtime migrations.
+
+To enable schema history, install the CLI and initialize your branch:
+
+```bash
+curl -fsSL https://xata.io/install.sh | bash
+xata auth login
+xata init
+xata roll init
+```
+
+![Branch details/schema view](assets/images/getting-started/branch-details.png)
+
+## 6. Create a development branch with the CLI
+
+List your branches:
+
+```bash
+xata branch list
+```
+
+Grab the `main` branch ID from the list and create a new branch from `main`:
+
+```bash
+ xata branch create --name dev --parent-branch [main branch id]
+```
+
+## 7. Modify your schema from the terminal
+
+You can make schema changes directly in the Xata query view, but here we'll show how to do it from your terminal using `psql` for a more hands-on workflow.
+
+If you don't have `psql` installed:
+
+**macOS:**
+```bash
+brew install libpq
+echo 'export PATH="/opt/homebrew/opt/libpq/bin:$PATH"' >> ~/.zshrc
+source ~/.zshrc
+```
+
+**Linux:**
```

Comment:
Better say Ubuntu here, since that command won't work on Redhat based distro. Also, I would be explicit with version 17:

```
sudo apt install postgresql-client-17
```